doc/src/pair_local_density.txt

```diff
@@ -0,0 +1,207 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Commands_all.html)
+
+:line
+
+pair_style local/density command :h3
+
+[Syntax:]
+
+pair_style style arg :pre
+
+style = {local/density}
+arg = name of file containing tabulated values of local density and the potential :ul
+
+[Examples:]
+
+pair_style local/density benzene_water.localdensity.table :pre
+
+pair_style hybrid/overlay table spline 500 local/density 
+pair_coeff * * local/density  benzene_water.localdensity.table :pre
+
+[Description:]
+
+The local density (LD) potential is a new potential style that is, in some 
```

Comment:
I suggest removing "new" b/c this doc page could exist for N years.
Just cite the papers and it will be clear when it came about.

Reply:
done.